ingest/cli_parser.py

```diff
@@ -106,6 +106,13 @@ def create_parser():
         "--profile-memory", action="store_true", help=profile_memory_text
     )
 
+    parser.add_argument(
+        "--user-metrics-uuid",
+        required=False,
+        type=is_valid_uuid,
```

Comment:
I don't feel strongly about this, but, if it were me, I wouldn't validate that this is a UUID.  We're essentially just saying "give me a string that I'll pass along for you to a third-party".  Since ingest-pipeline itself doesn't care about the string itself, requiring it to be a uuid can just make the code more brittle to change, and unduly sensitive to how various languages format guids